crates/cxx-qt-lib/src/types/qrect.rs

```diff
@@ -73,6 +74,19 @@ impl Default for QRect {
     }
 }
 
+impl fmt::Display for QRect {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
should we use Qt for the display format ? As most Qt types have `QDataStream &operator<<(QDataStream &stream, const T &point)`  eg i wonder if this would work ... 

Add the following to  `crates/cxx-qt-lib-headers/include/common.h` ...
```cpp
template<typename T>
QString
display(const T& value)
{
  return QString() << value;
}
```

Then add that to the bridge
```rust
        #[doc(hidden)]
        #[rust_name = "qrect_display"]
        fn display(value: &QRect) -> QString;
```

Then in the `fmt` methods do
```rust
write!(f, "{}", qrect_display(self))
```

Reply:
That's a good idea.
The simple `QString() << value;` that you proposed doesn't quite work, but I was able to get it working by constructing a `QDebug` object that writes into a `QString` and streaming (`<<`) into that instead.